DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/DelegateGroupsController.cs

```diff
@@ -87,129 +87,106 @@ public IActionResult AllDelegateGroups()
 

```

Comment:
We're going to have some merge conflicts in this file

Reply:
Sorted those.